src/components/checkbox/CheckboxField.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import * as React from 'react';
+import type { FieldProps } from 'formik';
+
+import CheckboxPrimitive from './Checkbox';
+import type { CheckboxProps } from './Checkbox';
+
+type Props = CheckboxProps & FieldProps;
+
+const CheckboxField = ({ field, ...rest }: Props) => {
+    const { value } = field;
+    return <CheckboxPrimitive {...field} {...rest} isChecked={!!value} />;
```

Comment:
Would it be worth forwarding refs from the field to the inner primitive? If we need to call `input.focus` somewhere, for example?